Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -192,52 +225,91 @@ public void TestDependencyCollectionDiagnosticSourceWithParentActivity()
 
         [TestMethod]
         [Timeout(5000)]
-        public void TestDependencyCollectionFailedRequestDiagnosticSource()
+        public async Task TestDependencyCollectionFailedRequestDiagnosticSource()
```

Comment:
Rename test to indicate the failure is at DNS Resolution itself , compared to failure due to status code > 400